### ISSUE TITLE
Pointers to CDN missing

### ISSUE THREAD

**Issue by ydogandjiev** [MicrosoftDocs/msteams-docs#8198]
The Teams Client SDK can be consumed via a set of CDN endpoints. It looks like all the references to these have been removed from this page. Can you please re-add them?

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 4ae8187e-a7e4-4b5a-6f38-da6bfa30f2d2
* Version Independent ID: 4ae8187e-a7e4-4b5a-6f38-da6bfa30f2d2
* Content: [Teams JavaScript client library SDK - Teams](https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/using-teams-client-library?tabs=javascript%2Cmanifest-teams-toolkit)
* Content Source: [msteams-platform/tabs/how-to/using-teams-client-library.md](https://github.com/MicrosoftDocs/msteams-docs/blob/main/msteams-platform/tabs/how-to/using-teams-client-library.md)
* Service: **msteams**
* GitHub Login: @heath-hamilton
* Microsoft Alias: **surbhigupta**

**Comment by ChetanSharma-msft** [comment]
Hello @ydogandjiev - We will look into it and will redirect to doc team accordingly.

**Comment by sayali-MSFT** [comment]
@surbhigupta12 -Could you please help us to update the document.
Update the below sentence -

The TeamsJS client is distributed as an [npm package](https://npmjs.com/package/@microsoft/teams-js/) and can be installed using npm or yarn or [CDN](https://www.npmjs.com/package/@microsoft/teams-js).

Also add the 3rd option in table section as CDN
Add the below link in that section-
```
<!-- Microsoft Teams JavaScript API ([via CDN](https://www.npmjs.com/package/@microsoft/teams-js)) -->
<script
  src="https://res.cdn.office.net/teams-js/2.10.1/js/MicrosoftTeams.min.js"
  integrity="sha384-Tc1x2xjzm4vRqLV4l416qPm+i7t8ujpxrioZEjRCNP7g/gQNDW41TB1SpVkQxowY"
  crossorigin="anonymous"
></script>
```
Reference document- https://learn.microsoft.com/en-us/javascript/api/overview/msteams-client?view=msteams-client-js-latest&tabs=npm#finding-the-library-1

![Screenshot 2023-04-17 154206](https://user-images.githubusercontent.com/93527375/232455407-b80d93cd-73f2-4c28-80ad-b0bd5824e572.png)

**Comment by erikadoyle** [comment]
Good catch @ydogandjiev! [Fixed now](https://learn.microsoft.com/en-us/javascript/api/overview/msteams-client?view=msteams-client-js-latest&tabs=npm#microsoft-teams-javascript-client-library).

**Comment by sayali-MSFT** [comment]
We are closing this issue for now, Please feel free to reopen it, if required.
